xchange-stream-coinbasepro/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.knowm.xchange</groupId>
         <artifactId>xchange-parent</artifactId>
-        <version>5.0.14-SNAPSHOT</version>
+        <version>5.0.12</version>
```

Comment:
This shouldn't be set like this

Reply:
Sorry, I made a temporary change and forgot to back out.

Now, Fixed.